packages/amplify-graphql-schema-generator/src/__tests__/__snapshots__/generate.test.ts.snap

```diff
@@ -145,6 +145,19 @@ type Story @refersTo(name: \\"stories\\") @model {
 "
 `;
 
+exports[`generate graphqlSchemaFromSQLSchema creates graphql schema from "postgres" "serial" schema 1`] = `
+"input AMPLIFY {
+ engine: String = \\"postgres\\"  
+}
+
+
+type SerialTable @refersTo(name: \\"      serial_table\\") @model {
```

Comment:
Do we know why we have blank spaces before the table name for `refersTo` directive?

Reply:
Whitespace in my template string with the field info csv was causing this--resolved.